libs/utils/utils.js

```diff
@@ -253,6 +253,13 @@ export function getMetadata(name, doc = document) {
   return meta && meta.content;
 }
 
+(() => {
+  const mweb = getMetadata('mweb');
+  if (mweb === 'on') {
+    document.body.classList.add('mweb-enabled');
+  }
+})();
```

Comment:
We always strive to keep code in utils at a minimum, so why not something like:
```suggestion
(() => { if (getMetadata('mweb') === 'on') document.body.classList.add('mweb-enabled'); })();
```

Reply:
Done